snippets/go.json

```diff
@@ -62,7 +62,7 @@
 		},
 		"for range statement": {
 			"prefix": "forr",
-			"body": "for ${1:var} := range ${2:var} {\n\t$0\n}",
+			"body": "for ${1:_, }${2:var} := range ${3:var} {\n\t$0\n}",
```

Comment:
Since we always need 2 variables, one to hold the index, the other for the element shouldnt the comma be outside the placeholder?

Reply:
Like `{1:_},`

Reply:
The rationale for this is that:

1. I'd say in ~90% of cases you're iterating over slice values and you don't care about the first value, so you just tab through.
2. When you need to iterate over map keys, or channels, you can just press `<backspace>` to delete the whole first placeholder. If the comma is outside, you'd need to delete two things - the placeholder and the comma.
3. For the final case where you want to keep the key, you just type the name and one extra character (`,`).

That said, if you're unconvinced by my reasoning, I'm happy to change it :)

Reply:
Ah, good point